src/pages/publication/references_tab/components/PublisherRow.tsx

```diff
@@ -40,10 +36,9 @@ const StyledLevelText = styled.div`
   grid-area: level;
 `;
 
-const StyledButton = styled(Button)`
+const DeleteButton = styled(Button)`
```

Comment:
kunne laget egne komponenter av slike knapper. Men hvordan passer dette inn i mui-themes. Finner ikke et sted for å tilpasse kategori av knapper

Reply:
må undersøke dette nærmere tror jeg

Reply:
@elkebab har kanskje den erfaringen fra tidligere prosjekter ?

Reply:
Første tanke er egentlig at det burde holde med `color=primary|secondary|default`. Hvor primary er ok/neste/bekreft/etc, secodary er slett/fjern/etc, og default er info/etc.

Hvis dere syns det er litt rart kan jeg være med på å lage en egen liten Button collection, med egen DeleteButton, ConfirmButton, etc.  Men da syns jeg det er litt rart å bruke `<DeleteButton />` hvis det ikke er en faktisk slett-action, men bare en knapp som skal se lik ut igjen, så tror det blir vanskelig med god naming :/

Reply:
Da går vi for å bruke secondary til sletting/warnings for nå :-)

Reply:
... men deletebutton kan eksistere som en komponent som wrapper gitt tekst og icon i en secondary-button